orchestrator_service/src/main/resources/static/scripts/overview_scripts.js

```diff
@@ -0,0 +1,128 @@
+window.onload = function () {
+    drawUndefinedCircles(getUndefinedTransactionsData());
+    drawCategories(getCategoriesData());
+    let circles = document.querySelectorAll('.undefined-circle')
+    circles.forEach(function (circle) {
+        circle.addEventListener('dragstart', handleDragStart);
+        circle.addEventListener('dragend', handleDragEnd);
+    })
+    let categories = document.querySelectorAll('.category')
+    categories.forEach(function (category) {
+        category.addEventListener('dragenter', handleDragEnter)
+        category.addEventListener('dragleave', handleDragLeave)
+        category.addEventListener('dragover', handleDragOver);
+        category.addEventListener('drop', handleDrop);
+    })
+}
+
+const minUndefinedCircleSize = 100;
+const maxUndefinedCircleSize = 200;
+
+function getMaxSingleTransactionAmount() {
+    return 1000;
+}
```

Comment:
Я бы эту и след функцию переделал бы просто в переменные 
```suggestion
const maxSingleTransactionAmount = 1000;
```

Reply:
Изначально было так, но эта функция в будущем будет переделана на запрос с бэка, значение, которое она возвращает сейчас - просто заглушка